topic-operator/src/main/java/io/strimzi/operator/topic/KafkaStreamsConfiguration.java

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.topic;
+
+import io.apicurio.registry.streams.diservice.AsyncBiFunctionService;
+import io.apicurio.registry.streams.diservice.AsyncBiFunctionServiceGrpcLocalDispatcher;
+import io.apicurio.registry.streams.diservice.DefaultGrpcChannelProvider;
+import io.apicurio.registry.streams.diservice.DistributedAsyncBiFunctionService;
+import io.apicurio.registry.streams.diservice.LocalService;
+import io.apicurio.registry.streams.diservice.proto.AsyncBiFunctionServiceGrpc;
+import io.apicurio.registry.streams.distore.DistributedReadOnlyKeyValueStore;
+import io.apicurio.registry.streams.distore.FilterPredicate;
+import io.apicurio.registry.streams.distore.KeyValueSerde;
+import io.apicurio.registry.streams.distore.KeyValueStoreGrpcImplLocalDispatcher;
+import io.apicurio.registry.streams.distore.UnknownStatusDescriptionInterceptor;
+import io.apicurio.registry.streams.distore.proto.KeyValueStoreGrpc;
+import io.apicurio.registry.streams.utils.ForeachActionDispatcher;
+import io.apicurio.registry.streams.utils.Lifecycle;
+import io.apicurio.registry.streams.utils.LoggingStateRestoreListener;
+import io.apicurio.registry.utils.ConcurrentUtil;
+import io.apicurio.registry.utils.kafka.AsyncProducer;
+import io.apicurio.registry.utils.kafka.ProducerActions;
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.ServerInterceptors;
+import io.grpc.Status;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.Topology;
+import org.apache.kafka.streams.errors.InvalidStateStoreException;
+import org.apache.kafka.streams.state.HostInfo;
+import org.apache.kafka.streams.state.ReadOnlyKeyValueStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * Configure all things needed for KafkaStreamsTopicStore
+ */
+public class KafkaStreamsConfiguration {
+    private static final Logger log = LoggerFactory.getLogger(KafkaStreamsConfiguration.class);
+
+    private final List<Object> closeables = new ArrayList<>();
+
+    KafkaStreams streams; // exposed for test purposes
```

Comment:
Conventionally, in the rest of the code, fields have been annotated like so:
```
/* test */ KafkaStreams streams;
```

Again not a big deal, but it makes it slightly easier to navigate and understand when the repo is consistent